src/main/java/core/basesyntax/animals/Penguin.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.animals;
+
+import core.basesyntax.types.Bird;
+
+public class Penguin extends Bird {
```

Comment:
```suggestion
public class Penguin extends Bird implements Swimmable {
```
пінгвін має вміти плавати

Reply:
Виправив. Дякую!